src/reyna/Preferences.cs

```diff
@@ -78,6 +164,85 @@ internal static void ResetCellularDataBlackout()
             DeleteRegistryValue("DataBlackout:To");
         }
 
+        internal static void SetWlanBlackoutRange(string range)
```

Comment:
I prefer to have a method for resetting the value.
it make it clear.
as with ResetWwanBlackoutRange and ResetRoamingBlackout.
@Youhana-Hana